int32 motor_num

float32[] pos_p_gain_throttle
float32[] pos_i_gain_throttle
float32[] pos_d_gain_throttle

float32[] pos_p_gain_yaw
float32[] pos_i_gain_yaw
float32[] pos_d_gain_yaw